### PR TITLE
Add COC

### PR DESCRIPTION
**Pull request by jamiehannaford**

### CODE REVIEW

File: code_of_conduct.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Kubernetes Community Code of Conduct
```
Comment (floreks) [inline code_of_conduct.md]: Shouldn't it be `Kubernetes Dashboard...`?

File: code_of_conduct.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+Kubernetes follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
```
Comment (floreks) [inline code_of_conduct.md]: Same here.

File: code_of_conduct.md (hunk 1 above)
Comment (jamiehannaford) [inline code_of_conduct.md]: Since it's a project-wide CoC I thought `Kubernetes` would be enough, but I can personalise if you think it's better :)

Comment (floreks) [inline code_of_conduct.md]: I guess this doesn't make much difference. I see that other projects in k8s org also use the same sentence. We can keep it this way then.

**Pull request merged**